src/liger_kernel/ops/rms_norm.py

```diff
@@ -236,33 +254,40 @@ def rms_norm_forward(X, W, eps, offset, casting_mode):
 def rms_norm_backward(dY, X, W, RSTD, offset, casting_mode, BLOCK_SIZE, num_warps):
     shape = dY.shape
     dim = shape[-1]
-    dY = dY.view(-1, dim)
+    dY = dY.view(-1, dim).contiguous()
```

Comment:
in most of the case this is contiguous right?

Reply:
Oh my bad, I simply forgot to remove it after testing around.